wordpress-mobile/gutenberg-mobile
# Merge blocks of different types

### Description:

This PR adds the ability of merging blocks of different types.

Moves #167 forward.

### Related PRs:

Gutenberg side: https://github.com/WordPress/gutenberg/pull/11294 )(already merged!)

### Testing:

1. Place the caret at the beginning of the first para block in the demo content.
2. Press backspace.
3. Make sure the resulting block is as expected (Heading block) and refreshed properly.
